README.md

```diff
@@ -66,22 +66,37 @@ const status = response.getStatus();
 const message = response.getMessage();
 ```
 
+### Register functions
+Use the `registerFunction` function to register a function.
+```
+const response = await clientService.registerFunction('functionId, 'com.example.function.functionName', functionUint8Array);
+const status = response.getStatus();
+const message = response.getMessage();
+```
+
 ### List registered contracts
-Use listContracts function to list all registered contracts.
+Use `listContracts` function to list all registered contracts.
 ```
 const response = await clientService.listContracts();
 const message = response.getMessage();
 const contracts = JSON.parse(message);
 ```
 
 ### Execute a contract
-Use executeContract function to execute a registered contract.
+Use `executeContract` function to execute a registered contract.
```

Comment:
```suggestion
Use `executeContract` function to execute a registered contract. It will also execute a function if `_functions_` is given in the argument. 
```